File: 2240-number-of-ways-to-buy-pens-and-pencils/2240-number-of-ways-to-buy-pens-and-pencils.cs
public class Solution {
    public long WaysToBuyPensPencils(int total, int cost1, int cost2) {
        long res=0;
        while(total >= 0){
            res += ((total/cost2)+1);
            total -= cost1;
        }
        return res;
    }
}

File: 2240-number-of-ways-to-buy-pens-and-pencils/2240-number-of-ways-to-buy-pens-and-pencils.js
/**
 * @param {number} total
 * @param {number} cost1
 * @param {number} cost2
 * @return {number}
 */
var waysToBuyPensPencils = function(total, cost1, cost2) {
    var res = 0;
    while(total >= 0){
        res += Math.trunc((total/cost2)+1);
        total -= cost1;
    }
    return (res);
};